### ISSUE TITLE
Any Video building it on VS2019 for GNU-EFI ?

### ISSUE THREAD

**Issue by gomidas** [Bareflank/standalone_cxx#3]
I want to add c++ feature to my gnu-efi project on vs2019. I do not have any idea that how to do it with vs2019. Is there a video tutorial for vs2019?

**Comment by rianquinn** [comment]
No. At the moment, this doesn't compile on Windows for UEFI. You need to compile using Linux.
